Modules/CalcOffence-3_0.lua

```diff
@@ -1172,13 +1172,96 @@ function calcs.offence(env, actor, activeSkill)
 		output.FistOfWarHitMultiplier = skillModList:Sum("BASE", cfg, "FistOfWarHitMultiplier") / 100
 		output.FistOfWarAilmentMultiplier = 1 + skillModList:Sum("BASE", cfg, "FistOfWarAilmentMultiplier") / 100
 		if output.FistOfWarCooldown ~= 0 then
-			output.FistOfWarHitEffect = 1 + output.FistOfWarHitMultiplier / (output.FistOfWarCooldown * output.Speed)
-			output.FistOfWarAilmentEffect = 1 + output.FistOfWarAilmentMultiplier / (output.FistOfWarCooldown * output.Speed)
+			output.FistOfWarHitEffect = 1 + output.FistOfWarHitMultiplier / m_max(output.FistOfWarCooldown * output.Speed, 1.0)
+			output.FistOfWarAilmentEffect = 1 + output.FistOfWarAilmentMultiplier / m_max(output.FistOfWarCooldown * output.Speed, 1.0)
 		else
 			output.FistOfWarHitEffect = 1
 			output.FistOfWarAilmentEffect = 1
 		end
 
+		-- Calculate Exerted Attacks
+		output.extraExertions = skillModList:Sum("BASE", nil, "ExtraExertedAttacks") or 0
+		output.exertedUptime = 0
+		local bInstantWarcry = skillModList:Flag(nil, "InstantWarcry") or false
+		
+		output.maxSeismicExerts = skillModList:Sum("BASE", cfg, "SeismicExertedAttacks")
+		output.SeismicHitMultiplier = skillModList:Sum("BASE", cfg, "SeismicHitMultiplier") / 100
+		if output.maxSeismicExerts ~= 0 then
+			output.numSeismicExerts = output.maxSeismicExerts + output.extraExertions
+			for _, value in ipairs(env.auxSkillList) do
+				if value.skillCfg.skillName == "Seismic Cry" then
+					-- get cooldown
+					local gemDefaultCooldown = value.skillData.cooldown
+					output.SeismicCryCooldown = gemDefaultCooldown / (1 + skillModList:Sum("INC", value.skillCfg, "CooldownRecovery") / 100)
```

Comment:
Needs to take into account the base added cooldown of Urgen Orders Support with `skillModList:Sum("BASE", skillCfg, "CooldownRecovery")`